### DISCUSSION TITLE
无法安装最新版本

### DISCUSSION THREAD

**Discussion by thouser8** [ywmoyue/biliuwp-lite#714] [Q&A] [answered]
### Bug 描述

执行脚本安装时报错
![efb92a0b83a65f1d74bfccc60eae2501](https://github.com/user-attachments/assets/249e57ce-48db-433c-8039-e4fd8e4cd364)

### 操作步骤

```plain text
按照安装教程执行无法安装
```

### 期望结果

找到安装问题成因

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 其它系统版本

Windows 11 24H2 26100.994

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
是否已卸载旧版本应用并重启系统

**Comment by thouser8** [comment]
> 是否已卸载旧版本应用并重启系统

是的

**Comment by ywmoyue** [comment]
> > 是否已卸载旧版本应用并重启系统
> 
> 是的

试试直接双击msix进行安装呢？这样缺失的依赖会从应用商店下载

**Comment by thouser8** [comment]
> > > 是否已卸载旧版本应用并重启系统
> > 
> > 
> > 是的
> 
> 试试直接双击msix进行安装呢？这样缺失的依赖会从应用商店下载

![QQ_1721446396028](https://github.com/user-attachments/assets/b1e949fb-3449-4e16-b85f-89a1de084307)
同样不行

**Comment by ywmoyue** [comment]
奇怪，4.7.0的x64安装包不应该依赖.net.native包呀，为什么你的依赖文件夹中有这个，方便截图一下你的这个x64文件夹吗
![image](https://github.com/user-attachments/assets/e2637ac0-4e2d-438f-af11-b0f246a96967)

**Comment by ywmoyue** [comment]
如果你的解压路径和旧版本是同一个路径的话，你需要先删除原文件夹中的内容，因为4.7.0和4.6.x依赖不同

**Comment by thouser8** [comment]
> 奇怪，4.7.0的x64安装包不应该依赖.net.native包呀，为什么你的依赖文件夹中有这个，方便截图一下你的这个x64文件夹吗 ![image](https://private-user-images.githubusercontent.com/28388300/350659919-e2637ac0-4e2d-438f-af11-b0f246a96967.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0NDc1ODAsIm5iZiI6MTcyMTQ0NzI4MCwicGF0aCI6Ii8yODM4ODMwMC8zNTA2NTk5MTktZTI2MzdhYzAtNGUyZC00MzhmLWFmMTEtYjBmMjQ2YTk2OTY3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIwVDAzNDgwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRkN2VkZWUyYzJlNWVhODg4YzE5NDExNjhhNTQ4MTcwMGQwZWM1ZWI3ZDY4ZWYyMzI5NDM2MTU5NTEwNDFkMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.4cmZQfDlM-_W7W70ujMMonQzdj0Ln184O-XwQB__clg)
![QQ_1721447334651](https://github.com/user-attachments/assets/6e638da7-f9e7-47b7-9668-b8f5af342067)

**Comment by ywmoyue** [comment]
> 如果你的解压路径和旧版本是同一个路径的话，你需要先删除原文件夹中的内容，因为4.7.0和4.6.x依赖不同

![image](https://github.com/user-attachments/assets/46362e78-c94c-4135-8863-1ee4e4218548)

4.7.0的压缩包里面没有.net.native包，你看是否先处理下旧版本的解压内容

**Comment by thouser8** [comment]
> 如果你的解压路径和旧版本是同一个路径的话，你需要先删除原文件夹中的内容，因为4.7.0和4.6.x依赖不同

![QQ_1721447904602](https://github.com/user-attachments/assets/1ace832b-8941-4c5a-a64d-65f397004afa)
按照你说的删干净了旧版本的文件重新安装

**Comment by ywmoyue** [comment]
> > 如果你的解压路径和旧版本是同一个路径的话，你需要先删除原文件夹中的内容，因为4.7.0和4.6.x依赖不同
> 
> ![QQ_1721447904602](https://private-user-images.githubusercontent.com/83090571/350661579-1ace832b-8941-4c5a-a64d-65f397004afa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0NDgyNDcsIm5iZiI6MTcyMTQ0Nzk0NywicGF0aCI6Ii84MzA5MDU3MS8zNTA2NjE1NzktMWFjZTgzMmItODk0MS00YzVhLWE2NGQtNjVmMzk3MDA0YWZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIwVDAzNTkwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI2YzY3ZGZiYmJmMjAzYjBhZmVmNTU3ODcyODYzYzcxYThmNDM3ZTg5NTMxNzIzODUxZDY5MzEwZTRiMmM5YzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wfLxQ1xW6DKTf8wNXjAEoiOoCmUBPWV9EOwgM7oV0a8) 按照你说的删干净了旧版本的文件重新安装

这时就需要重启系统了，报错是资源正在使用中，我猜前面的安装进程还没关掉，我自己遇到这个错误是重启解决的

**Comment by thouser8** [comment]
> > > 如果你的解压路径和旧版本是同一个路径的话，你需要先删除原文件夹中的内容，因为4.7.0和4.6.x依赖不同
> > 
> > 
> > ![QQ_1721447904602](https://private-user-images.githubusercontent.com/83090571/350661579-1ace832b-8941-4c5a-a64d-65f397004afa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE0NDgyNDcsIm5iZiI6MTcyMTQ0Nzk0NywicGF0aCI6Ii84MzA5MDU3MS8zNTA2NjE1NzktMWFjZTgzMmItODk0MS00YzVhLWE2NGQtNjVmMzk3MDA0YWZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzIwVDAzNTkwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI2YzY3ZGZiYmJmMjAzYjBhZmVmNTU3ODcyODYzYzcxYThmNDM3ZTg5NTMxNzIzODUxZDY5MzEwZTRiMmM5YzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wfLxQ1xW6DKTf8wNXjAEoiOoCmUBPWV9EOwgM7oV0a8) 按照你说的删干净了旧版本的文件重新安装
> 
> 这时就需要重启系统了，报错是资源正在使用中，我猜前面的安装进程还没关掉，我自己遇到这个错误是重启解决的

但是我重启了三次还是一样的问题🙁

**Comment by ywmoyue** [comment]
@thouser8  删除报错的这个Microsoft.VCLibs.x64.14.00.Desktop.appx 再试试

**Comment by thouser8** [comment, marked as answer]
> @thouser8 删除报错的这个Microsoft.VCLibs.x64.14.00.Desktop.appx 再试试

把Microsoft.VCLibs.x64.14.00.Desktop.appx Microsoft.VCLibs.x64.14.00.appx这两个文件删掉后终于是安装成功了，感谢
